app/component/itinerary.scss

```diff
@@ -191,6 +191,158 @@ $itinerary-tab-switch-height: 48px;
   }
 }
 
+.itinerary-co2-information {
+  &.mobile {
+    padding: 0 15px;
+    .divider-bottom {
+      width: 365px;
+    }
+  }
+}
+
+.itinerary-co2-line {
+  position: relative;
+
+  .divider-top,
+  .divider-bottom {
+    position: relative;
+    width: 400px;
+    height: 1px;
+    left: -5px;
+    border-top: 1px solid #dddddd;
+    box-sizing: border-box;
+
+    @media print {
+      border: none;
+    }
+  }
+
+  .co2-description-container {
+    display: flex;
+    gap: 20.01px;
+    justify-content: space-between;
+    align-items: start;
+    margin-top: 5px;
+    margin-bottom: 5px;
+
+    .icon-container {
+      .icon {
+        &.co2-leaf {
+          height: 25.06px;
+          width: 25.6px;
+          margin-top: 5px;
+          margin-left: 5px;
+        }
+      }
+    }
+
+    .itinerary-co2-description {
+      width: 275px;
+      left: 70px;
+      top: 528px;
+      &.simple {
+        width: auto;
+      }
+      font-family: $font-family;
+      font-style: normal;
+      font-weight: 325;
+      font-size: 15px;
+      line-height: 20px;
+      letter-spacing: -0.03em;
+      display: flex;
+      flex-direction: column;
+      color: #666666;
+    }
+
+  }
+
+  .co2-container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 11px 10px 10px 15px;
+    &.mobile {
+      flex-direction: row;
+      margin: 11px 10px 10px 6px
+    }
+    @include min-width(tablet) {
+      margin-right: 15px;
+    }
+
+    .co2-title-container {
+      display: flex;
+      gap: 11.34px;
+      justify-content: space-between;
+      align-items: center;
+
+      .icon-container {
+        .icon {
+          &.co2-leaf {
+            height: 13.91px;
+            width: 14.22px;
+          }
+        }
+      }
+
+      .itinerary-co2-title {
+        width: 146px;
+        height: 18px;
+        left: 48px;
+        top: 528px;
+
+        font-family: $font-family;
+        font-style: normal;
+        font-weight: 325;
+        font-size: 15px;
+        line-height: 18px;
+        /* identical to box height */
+        display: flex;
+        align-items: center;
+        letter-spacing: -0.03em;
+
+        /* Grayscale/HSL Black 75 */
```

Comment:
Seeing something HSL specific outside of configuration makes me wonder if something should belong to configuration. On the other hand, seems like we have used this color in a lot of places and not just for HSL. Maybe just remove the HSL reference from the comment or the whole comment :D.